File: PythonPrograms/2_Patterns/StarPatterns/star.py
# star pyramid patterns
# half pyramid patterns
# Traingle patterns
# Hourglass patterns
# Diamond patterns
# Inverted pyramid patterns

# 1. pyramid
# *
# **
# ***
# ****
# *****

def pattern(n,symbol):
    space = n
    for i in range(0,n):
        for j in range(0,space-1):
            print(end=" ")
        

        for j in range(0,i+1):
            print(symbol,"",end="")
        print("\r") 
        space = space -1



rows = int(input("rows : "))
symbol = input("symbol : ")

print("\n-------------\n")
pattern(rows,symbol)               

File: Patterns/StarPatterns/star.cpp
#include<iostream>
using namespace std;

int main()
{
    int n = 15;
    int space = n;
    for(int i = 0; i<n; i++)
    {
        for(int j=0; j<space-1;j++)
        cout<<" ";

        for(int j =0; j<i+1; j++)
        cout<<"* ";
        space--;
        printf("\n");

    }

    return 0;
}